support-figma/extended-layout-plugin/src/ui.html

```diff
@@ -759,63 +798,141 @@
   function createInputStringTable(tableContainerId, data) {
     let {table, tableContainer} = createTable(tableContainerId);
 
-    const headers = ["String Resource Name", "Translatable", "Text"];
+    const headers = ["String Resource Name", "Translatable", "Text", "Extras"];
     createTableHeader(table, headers);
 
     for (const rowData of data) {
       var cells = Object.values(rowData);
       const row = document.createElement("tr");
       for (const cellData of cells) {
-        createCell(row, cellData, false);
+        row.appendChild(createCell(cellData, false));
       }
       table.appendChild(row);
     }
 
     tableContainer.appendChild(table);
   }
 
+  function createExtrasCell(stringRes) {
+    const td = document.createElement("td");
+    let extras = stringRes["extras"];
+    if (!extras) {
+      extras = {"charlimit": "NONE", "description": ""};
+      stringRes["extras"] = extras;
+    }
+    const textBlock = document.createElement("div");
+    textBlock.textContent = JSON.stringify(extras);
+    td.appendChild(textBlock);
+    if (stringRes["textNodes"] != null) {
+      const iconButton = document.createElement("div");
+      iconButton.className = "icon-button";
+      const svgElement = document.createElementNS("http://www.w3.org/2000/svg", "svg");
+      const useElement = document.createElementNS("http://www.w3.org/2000/svg", "use");
+      useElement.setAttributeNS("http://www.w3.org/1999/xlink", "xlink:href", "#icon-edit");
+      svgElement.appendChild(useElement);
+      iconButton.appendChild(svgElement);
+      td.appendChild(iconButton);
+
+      const form = document.createElement("form");
+      form.style.display="none";
+
+      const charlimitInput = document.createElement("input");
+      charlimitInput.type = "number";
+      charlimitInput.min = "0";
+      charlimitInput.max = "200";
+      charlimitInput.step = "5";
```

Comment:
I don't know much about Android localization, but is there any reason this is 5? What if I want to set the limit to something not divisible by 5?

Reply:
Per discussion offline. fine with 5s or 10s.